Library/Formula/ansible.rb

```diff
@@ -204,6 +204,14 @@ class Ansible < Formula
     sha256 "3e95445c1db500a344079a47b171c45ef18f57d188dffdb0e4165c71bea8eb3d"
   end
 
+	#
+	# apache libcloud (for Google GCE cupport)
+	#
```

Comment:
@rmarsagoogle No need to, you can update this pull-request by pushing new commits to it.